src/globals/index.ts

```diff
@@ -82,11 +82,15 @@ export const EMPTY_TOKEN: DefaultTokenObject = {
 // Network token list hashes (latest versions)
 export const TESTING_TOKEN_LIST_HASH = 'QmXgUiWTumXghNuLk3vAypVeL4ycVkNKhrtWfvFHoQTJAM'
 
-export const RINKEBY_TOKEN_LIST_HASH = 'QmQN7tJSVTbREv7Gd6FWZhno9RPRcFNDNkcgni6xe6wDBo'
+export const RINKEBY_TOKEN_LIST_HASH = process.env.NODE_ENV === 'production' ?
+  'QmRLt9FQ8gu1Zs9wJEXtBJDagnTHFWbYjrUDuu1mZG8TCw' : 'QmbQfGNBEwdJU7rvHQC4ef3UxbegLUL6ZPH7E1Jtxj3zmo'
```

Comment:
why the differentiation here? the `development` tokens on Rinkeby are OMG and OMG Test tokens + the rest are same. Why not add DAI and GEN in dev?